src/components/atom/Circle/Circle.tsx

```diff
@@ -0,0 +1,15 @@
+import "./style.css";
+
+type recivedNumbers = {
```

Comment:
It should be `receivedNumbers` but I think `PropsType` should be a clearer name for a type.